packages/design-system/src/components/menu/list/defaultListItem.tsx

```diff
@@ -37,34 +37,30 @@ const ActiveIcon = styled(CheckmarkSmall)`
   transform: translateY(-50%);
 `;
 
-const DefaultListItem = ({ option, isSelected, ...rest }, ref) => (
-  <ListItem
-    {...rest}
-    ref={ref}
-    isSelected={isSelected}
-    disabled={option.disabled}
-    aria-disabled={option.disabled}
-  >
-    {isSelected && (
-      <ActiveIcon
-        data-testid={'dropdownMenuItem_active_icon'}
-        aria-label={__('Selected', 'web-stories')}
-        width={32}
-        height={32}
-      />
-    )}
-    <ListItemDisplayText
-      forwardedAs="span"
-      size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
+const DefaultListItem = forwardRef(function DefaultListItem(
+  { option, isSelected, ...rest }: DefaultListItemProps,
+  ref: ForwardedRef<HTMLLIElement>
```

Comment:
`HTMLLIElement`? Because of `value`?

Reply:
Because it's a list item? Not sure I understand the question?